packs/okta.yml

```diff
@@ -8,6 +8,7 @@ PackDefinition:
     - Okta.APIKeyRevoked
     - Okta.BruteForceLogins
     - Okta.GeographicallyImprobableAccess
+    - Okta.Support.Access
```

Comment:
I think we might need to add the data model too

Reply:
Added